libs/langchain-google-vertexai-web/README.md

```diff
@@ -0,0 +1,40 @@
+# LangChain google-vertexai-web
+
+This package contains resources to access Google AI/ML models
+and other Google services via Vertex AI. Authorization to these
+services use either an API Key or service account credentials
+that are included in an environment variable.
+
+If you are running this on the Google Cloud Platform, or in a way
+where service account credentials can be stored on a file system,
+consider using the @langchain/google-vertexai
+package *instead*. You do not need to use both packages. See the
+section on **Authorization** below.
+
+
+## Installation
+
+```bash
+$ yarn add @langchain/google-vertexai-web
+```
+
+
+## Authorization
+
+Authorization is either done through the use of an API Key, if it is
```

Comment:
API Keys are not supported with Vertex AI.
(Tho the underlying class will support it.)